core/src/trezor/utils.py

```diff
@@ -121,11 +121,10 @@ def presize_module(modname: str, size: int) -> None:
     symbols defined in the file itself, but also on the number of submodules that will
     be inserted into the module's namespace.
     """
+    from trezorutils import presize_module
```

Comment:
could we just `import presize_module` on top level and remove this function altogether? we'd need to change the call sites to pass the module object itself instead of its name, but that shouldn't be a problem, right?

Reply:
Thanks! Fixed in https://github.com/trezor/trezor-firmware/pull/4934/commits/4a1fe51008281e304ffc809f80ee0a92eff4b709.